src/custom/constants/index.ts

```diff
@@ -101,7 +101,7 @@ export const RAW_CODE_LINK = 'https://raw.githubusercontent.com/' + GITHUB_REPOS
 export const DOCS_LINK = 'https://docs.cow.fi'
 export const CONTRACTS_CODE_LINK = 'https://github.com/cowprotocol/contracts'
 export const DISCORD_LINK = 'https://discord.com/invite/cowprotocol'
-export const DUNE_DASHBOARD_LINK = 'https://duneanalytics.com/gnosis.protocol/Gnosis-Protocol-V2'
+export const DUNE_DASHBOARD_LINK = 'https://dune.com/gnosis.protocol/Gnosis-Protocol-V2'
```

Comment:
If we are updating the link, better use the one that points to COW Protocol rather than gp?

I got the general https://dune.com/gnosis.protocol/CowSwap, but even this is still under gnosis.protocol.

Better update it another time then

Reply:
Will review this URL to make sure it's the right one.